audiocall/src/main/java/com/voximplant/demos/kotlin/audio_call/services/AudioCallManager.kt

```diff
@@ -339,6 +399,27 @@ class AudioCallManager(
             startActivity(appContext, it, null)
         }
     }
+
+    private fun playProgressTone() {
+        callProgressToneFile = Voximplant.createAudioFile(appContext, R.raw.call_progress_tone, AudioFileUsage.IN_CALL)
+        callProgressToneFile?.play(true)
+    }
+
+    private fun stopProgressTone() {
+        callProgressToneFile?.stop(false)
+        callProgressToneFile?.release()
+    }
+
+    private fun playConnectedTone() {
+        callConnectedToneFile = Voximplant.createAudioFile(appContext, R.raw.call_connected_tone, AudioFileUsage.IN_CALL)
+        callConnectedToneFile?.play(false)
```

Comment:
It is required to call IAudioFile.release() after the file is played, otherwise MediaPlayer resources will leak.

Another option is to create the audio file on AudioCallManager initialization and reuse it.

The same for `callFailedToneFile`.